ui/ui-components/pages/app/[id]/refresh.tsx

```diff
@@ -88,8 +88,26 @@ export default function Page(props) {
     );
     if (response?.appRefreshQuery) {
       setLoading(false);
-      successHandler.set({ message: "App Refresh Query Updated" });
       setAppRefreshQuery(response.appRefreshQuery);
+
+      try {
+        const response: Actions.AppRefreshQueryQuery = await execApi(
+          "post",
+          `/appRefreshQuery/${appRefreshQuery.id}/query`
+        );
+        if (response?.valueUpdated == true) {
+          successHandler.set({
+            message: `App Refresh Query updated. Returned ${response.appRefreshQuery.value}. Enqueueing Schedules.`,
+          });
+        } else {
+          successHandler.set({
+            message: `App Refresh Query updated. Query returned ${response.appRefreshQuery.value}. No schedules enqueued.`,
+          });
+        }
+        setAppRefreshQuery(response.appRefreshQuery);
+      } finally {
+        setLoading(false);
+      }
```

Comment:
This is new too - looks like after editing the query, you will try to run it now.  Makes sense!